spec/rester/service/middleware/request_handler_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'support/test_service'
+
+module Rester
+  module Service::Middleware
+    RSpec.describe RequestHandler do
+      let(:service) {
+        Class.new(TestService).tap { |klass|
+          allow(klass).to receive(:name) { 'TestService' }
+        }
+      }
```

Comment:
Actually, can you test `RequestHandler` directly.  These tests are partially testing that `Service` uses the `RequestHandler` middleware by default.